src/custom/state/orders/reducer.ts

```diff
@@ -47,6 +47,10 @@ export type OrderLists = {
   fulfilled: PartialOrdersMap
   expired: PartialOrdersMap
   cancelled: PartialOrdersMap
+  creating: PartialOrdersMap
+  rejected: PartialOrdersMap
+  refunding: PartialOrdersMap
+  refunded: PartialOrdersMap
```

Comment:
I'm not sure we need all of those order buckets.
In any way, I think we have too many.
Would love to have a big refactor on all of this.
Don't want to start now though as it might delay us further at this point